### ISSUE TITLE
Can't install Plugins via Custom Plugin Registry

### ISSUE THREAD

**Issue by anthonma** [eclipse-che/che#14994]
### Describe the bug

I'm trying to install a plugin via my custom plugin registry.
If I try to do so, the worspace gets stuck when restarting after the installation has completed at:
````
Starting Init Plugin Broker
Unified Che Plugin Broker
````
A manual restart leads to that error:
````
Error: Failed to run the workspace: "Plugins installation process failed. Error: Plugin broking process for workspace workspacel4bfie9fhjfpwpwj failed with error: Downloading http://che-plugin-registry-che.192.168.99.100.nip.io/resources/github.com/microsoft/vscode-java-debug/releases/download/0.20.0/vscode-java-debug-0.20.0.vsix failed. Status code 404"
````

I have deployed it via the command given in the official docs:
````
NAMESPACE=<namespace-name> 
DOMAIN=<kubernetes-cluster-domain>
IMAGE="my-plugin-registry"
helm upgrade --install che-plugin-registry \
 --debug \
 --namespace $\{NAMESPACE} \
 --set global.ingressDomain=$\{DOMAIN} \
 --set chePluginRegistryImage=$\{IMAGE} \
 --set chePluginRegistryImagePullPolicy="IfNotPresent" \
 ./kubernetes/che-plugin-registry/
````
I just had to change $\{...} to ${...}, because with the \ I get a command not found error.

Since I'm facing that issue for a week and a half now, I have seen that the content of the plugin registry repo has changed, so I have tried it with the new version as well, but it didn't work either.
With this version I can't even deploy it.
I'm using this command to do so:
````
NAMESPACE="kube-che"
DOMAIN="$(minikube ip).nip.io"
helm upgrade --install che-plugin-registry \
    --debug \
    --namespace ${NAMESPACE} \
    --set global.ingressDomain=${DOMAIN} \
    ./kubernetes/che-plugin-registry/
````
and executing it leads to the following error:
````
Error: YAML parse error on che-plugin-registry-service/templates/configmap.yaml: error converting YAML to JSON: yaml: line 16: did not find expected node cntent
````
### Che version

 - [ ] latest
 - [ ] nightly
 - [x] other: 7.4.0 (with chectl)

### Runtime
   - [x] kubernetes (include output of `kubectl version`)
````
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.4", GitCommit:"67d2fcf276fcd9cf743ad4be9a9ef5828adc082f", GitTreeState:"clean", BuildDate:"2019-09-18T14:51:13Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.4", GitCommit:"67d2fcf276fcd9cf743ad4be9a9ef5828adc082f", GitTreeState:"clean", BuildDate:"2019-09-18T14:41:55Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```` 
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl (chectl server:start -p minikube)
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [x] my computer 
       - [x] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify) 
   - [ ] other: please specify

**Comment by JPinkney** [comment]
@anthonma as a temp fix I was able to get around this by removing {{ .Values.chePluginSidecarOverride.* }} so instead of:

```
kind: ConfigMap
apiVersion: v1
metadata:
  name: che-plugin-registry
data:
  CHE_SIDECAR_CONTAINERS_REGISTRY_URL: {{ .Values.chePluginSidecarOverride.url }}
  CHE_SIDECAR_CONTAINERS_REGISTRY_ORGANIZATION: {{ .Values.chePluginSidecarOverride.organization }}
  CHE_SIDECAR_CONTAINERS_REGISTRY_TAG: {{ .Values.chePluginSidecarOverride.tag }}}
```

I had:
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: che-plugin-registry
data:
  CHE_SIDECAR_CONTAINERS_REGISTRY_URL:
  CHE_SIDECAR_CONTAINERS_REGISTRY_ORGANIZATION:
  CHE_SIDECAR_CONTAINERS_REGISTRY_TAG:
```

with those removals, I was able to deploy the registry successfully. I know it's just a temp fix but hopefully, that helps at least with getting it deployed!

**Comment by amisevsk** [comment]
My bad -- it's a typo in the helm chart (three `}` instead of two on the last line). 

Should be fixed by PR https://github.com/eclipse/che-plugin-registry/pull/277 and similar for devfile registry: https://github.com/eclipse/che-devfile-registry/pull/134

**Comment by anthonma** [comment]
I can deploy the plugin registry now, but it seems that it builds and deploys the custom registry, because none of my changes are visible in the plugins view. I added a plugin which isn't showing up and for test purposes I changed the description of a existing plugin, but this change isn't shown as well.
To build I use the build.sh, which is in the registry folder without any parameters.
To deploy I use this command:
````
NAMESPACE="kube-che"
DOMAIN="$(minikube ip).nip.io"
helm upgrade --install che-plugin-registry \
    --debug \
    --namespace ${NAMESPACE} \
    --set global.ingressDomain=${DOMAIN} \
    ./kubernetes/che-plugin-registry/
```` 
I changed the last line to 
````
./deploy/kubernetes/che-plugin-registry
````
because of a path not found exception.
